<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>783</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
    <horstretch>1</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>CatRunner :: Send Mail With Template</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnLoadData">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>160</y>
      <width>140</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Data</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inpDataFilePath">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>40</y>
      <width>845</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border: solid 1px green; background-color: #ddd; color: #222222;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="acceptRichText">
     <bool>false</bool>
    </property>
    <property name="cursorWidth">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>path to excel data file...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnSendAll">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>834</x>
      <y>160</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: green;</string>
    </property>
    <property name="text">
     <string>Send All</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="dataList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>1000</width>
      <height>311</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>1</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderShowSortIndicator" stdset="0">
     <bool>true</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
   </widget>
   <widget class="QPushButton" name="btnBrowseData">
    <property name="geometry">
     <rect>
      <x>864</x>
      <y>40</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnViewData">
    <property name="geometry">
     <rect>
      <x>926</x>
      <y>40</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QListWidget" name="logList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>1000</width>
      <height>225</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="labelLog">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>515</y>
      <width>47</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Log:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDataList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Data List:</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="inpTemplateFilePath">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>100</y>
      <width>845</width>
      <height>30</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="styleSheet">
     <string notr="true">border: solid 1px green; background-color: #ddd; color: #222222;</string>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="cursorWidth">
     <number>0</number>
    </property>
    <property name="placeholderText">
     <string>path to template mail file...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnBrowseTemplate">
    <property name="geometry">
     <rect>
      <x>864</x>
      <y>100</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnViewTemplate">
    <property name="geometry">
     <rect>
      <x>926</x>
      <y>100</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelExcelFilePath">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>10</y>
      <width>155</width>
      <height>35</height>
     </rect>
    </property>
    <property name="text">
     <string>Excel File Path:</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelTemplateFilePath">
    <property name="geometry">
     <rect>
      <x>8</x>
      <y>75</y>
      <width>185</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Template Mail File Path:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
